src/components/TabTraining.vue

```diff
@@ -1,7 +1,26 @@
 <template>
   <div class="tab training">
     <h1>Training Options</h1>
+
+    We are using yaml files to set up the training configuration (see
+    config.yaml). Additional arguments given to the main script can be parsed
+    using one of the tools provided below:
+
     <h2>Argument Parser</h2>
+
+    <ul>
+      <li>
+        <a href="https://docs.python.org/3/library/argparse.html">Argparse</a> -
+        is python built-in tool to handler cmd arguments
+      </li>
+      <li>
+        <a
+          href="https://github.com/google/python-fire/blob/master/docs/guide.md"
+          >Python Fire</a
+        >
+        - is a helpful tool for developing and debugging python code
```

Comment:
this does not make much sense here.
Check out at least their description in the repo for inspirations
```
Python Fire is a library for automatically generating command line interfaces (CLIs) from absolutely any Python object. 
```